code/modules/vending/_vending.dm

```diff
@@ -1205,3 +1205,22 @@ GLOBAL_LIST_EMPTY(vending_products)
 		var/obj/item/I = target
 		I.custom_price = price
 		to_chat(user, "<span class='notice'>You set the price of [I] to [price] cr.</span>")
+
+/obj/machinery/vending/custom/greed //name and like decided by the spawn
+	custom_materials = list(/datum/material/gold = MINERAL_MATERIAL_AMOUNT * 5)
+	material_flags = MATERIAL_COLOR //it's grey anyway, let's bling out
+
+/obj/machinery/vending/custom/greed/Initialize(mapload)
+	. = ..()
+	//starts in a state where you can move it
+	panel_open = TRUE
+	anchored = FALSE
+	add_overlay("[initial(icon_state)]-panel")
+	//and references the deity
+	name = "[GLOB.deity]'s Consecrated Vendor"
+	desc = "A vending machine created by [GLOB.deity]."
+	slogan_list = list("[GLOB.deity] says: It's your divine right to buy!")
```

Comment:
"ONE DAY WHILE ANDY WAS says: It's your divine right to buy!"

Reply:
![image](https://user-images.githubusercontent.com/40974010/111883762-8d9faf00-897a-11eb-9d09-08c26a149115.png)